src/main/java/io/stargate/sgv2/jsonapi/api/model/command/impl/FindAndRerankCommand.java

```diff
@@ -48,6 +51,19 @@ public CommandName commandName() {
     return CommandName.FIND_AND_RERANK;
   }
 
+  @Override
+  public void addCommandFeaturesToCommandContext(CommandContext<?> context) {
+    context
+        .commandFeatures()
+        .addAll((sortClause != null) ? sortClause.getCommandFeatures() : CommandFeatures.EMPTY);
+    context
+        .commandFeatures()
+        .addAll(
```

Comment:
and same here; move `(options != null && options.hybridLimits() != null)` check into if-statement.